plugins/techdocs-backend/src/techdocs/stages/publish/local.ts

```diff
@@ -63,8 +66,10 @@ export class LocalPublish implements PublisherBase {
           reject(err);
         }
 
-        resolve({
-          remoteUrl: `http://localhost:7000/api/techdocs/static/docs/${entity.metadata.name}`,
+        this.discoveryApi.getBaseUrl('techdocs').then(techdocsApiUrl => {
+          resolve({
+            remoteUrl: `${techdocsApiUrl}/static/docs/${entity.metadata.name}`,
+          });
```

Comment:
maybe a bit of a nit but what if the getBaseUrl fails :) you should probably have a reject then